### DISCUSSION TITLE
blog/2023-04-12-boost-your-power-with-baseline-covariates/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#11] [Comments]
# blog/2023-04-12-boost-your-power-with-baseline-covariates/

Part 1 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-04-12-boost-your-power-with-baseline-covariates/

**Comment by moritzkoerber** [comment]
Cool post, great balance of accessibility and depth. One q: Wouldn't the 'classic' approach be a mixed-design ANOVA to check out the interaction effect?

**Reply by ASKurz** [reply]
I believe that would be a classic approach if you want to include the baseline measure within a literal analysis of variance, yes.
